hkmc2/shared/src/main/scala/hkmc2/semantics/Term.scala

```diff
@@ -297,7 +320,9 @@ final case class Param(flags: FldFlags, sym: LocalSymbol & NamedSymbol, sign: Op
   // def showDbg: Str = flags.showDbg + sym.name + ": " + sign.showDbg
   def showDbg: Str = flags.showDbg + sym + sign.fold("")(": " + _.showDbg)
 
-object FldFlags { val empty: FldFlags = FldFlags(false, false, false) }
+object FldFlags { 
```

Comment:
```suggestion
object FldFlags:
```

Reply:
I just reverted that change because there's no need to break this line. Do you want to keep the line break and change `{}` to `:`?

Reply:
Nope!